task/jib-gradle/0.4/README.md

```diff
@@ -0,0 +1,60 @@
+# Jib Gradle
+
+This Task builds Java/Kotlin/Groovy/Scala source into a container image using Google's [Jib](https://github.com/GoogleContainerTools/jib) tool.
+
+Jib works with [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin) and [Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin) projects, and this template is for Gradle projects.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/jib-gradle/0.4/jib-gradle.yaml
+```
+
+## Parameters
+
+- **BUILDER_IMAGE**: The location of the gradle builder image. (*default: gcr.io/cloud-builders/gradle@sha256:96d6343589392afd9eab8c4463ec899b8b1c972e7cd70678a70a4821c16eb4c9*)
+- **IMAGE**: Reference of the image gradle will produce.
+- **DIRECTORY**: The directory in the source repository where source should be found. (*default: .*)
+- **CACHE**: The name of the volume for caching Gradle artifacts, local Maven repository, and base image layers (*default: empty-dir-volume*)
+- **INSECUREREGISTRY**: Whether to allow insecure registry. (*default: "false"*)
+- **EXTRA_ARGS**: Provides extra arguments to the gradle jib build (*default: ""*)
+
+## Workspaces
+
+- **source**: A [Workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md) containing the source to build.
+
+## Results
+
+- **IMAGE_DIGEST**: The digest of the image just built.
```

Comment:
nit: IMAGE_URL is missing in results in the Readme

Reply:
5c13eeb3c7709e6b57310c166a08e0f7a2ec022b